pkgs/applications/editors/gnome-latex/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, wrapGAppsHook
+, tepl, amtk, gnome3, glib, pkgconfig, intltool, itstool, libxml2 }:
+let
+  version = "3.28.1";
+  pname = "gnome-latex";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${pname}-${version}.tar.xz";
+    sha256 = "1z481izrx057wraphnr82kxnpmmi8nvl7jswyylzm22kfs0mw402";
+  };
+
+  passthru.updateScript = gnome3.updateScript { packageName = pname; };
```

Comment:
Could you move this down before meta? We are trying to structure the expressions so that building instructions are first and meta things are last.